apps/tasks/lib/services/grpc_client_svc.dart

```diff
@@ -40,6 +40,7 @@ class GRPCClientService {
       "dapr-app-id": "task-svc",
     };
 
+    metaData["X-Organization"] = fallbackOrganizationId!;
     if (organizationId != null) {
       metaData["X-Organization"] = organizationId;
     }
```

Comment:
Line #43 makes Line #44-46 obsolete and vice versa. Line #43 as a `else`-Condition would be good. The caller still has the ability to define an organizationId.